src/vcpkg/commands.install.cpp

```diff
@@ -552,8 +551,6 @@ namespace vcpkg
                 perform_install_plan_action(args, paths, action, status_db, binary_cache, build_logs_recorder));
         }
 
-        compute_all_abis(paths, action_plan, var_provider, status_db);
-        binary_cache.prefetch(action_plan.install_actions);
```

Comment:
Now the `prefetch` is done before `Install::execute_plan`, but in line 544/543 the packages folder is removed.

Reply:
Those lines are only relevant for packages that are being removed.

Reply:
But if a package gets reinstalled with a different feature set it gets removed and then installed